pyproject.toml

```diff
@@ -21,8 +21,10 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.6"
 dataclasses = {version = ">=0.7 <0.9", python = "3.6"}
+docstring_parser = "^0.7.3"
```

Comment:
This should be made optional for users who do not use the Numpy style. Then the extra group would be called `numpy` :slightly_smiling_face:

Reply:
Agreed, done.